docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,157 @@
+# How to Contribute
```

Comment:
@JorisSchelfaut , @noahweasley : this is the core of the change, and what should be reviewed for clarity.